runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -491,6 +491,18 @@ TR_J9ServerVM::classHasBeenExtended(TR_OpaqueClassBlock *clazz)
    {
    if(!clazz)
       return false;
+
+   // Check CHTable first
```

Comment:
I am not sure whether we should check the `J9AccClassHasBeenOverridden` bit first and only then the CHTable.

Reply:
I am also thinking that we could update this bit if we find a subclass in the CHTable